apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.sql

```diff
@@ -6,3 +6,12 @@ FROM
 WHERE
   ci.course_id = $course_id
   AND ci.deleted_at IS NULL;
+
+-- BLOCK select_enrollment_count
+SELECT
+  COUNT(e.user_id)::integer AS enrollment_count
```

Comment:
nit: consider breaking this query and `apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.sql` into `apps/prairielearn/src/ee/models/enrollment.ts` to co-locate all the enrollment count queries.